### PR TITLE
added backend files

### PR DESCRIPTION
**Pull request by sammy-perry-kainos**

### CODE REVIEW

**Comment by sammy-perry-kainos** [comment]
PR covers adding all sql and backend code previously approved from other weeks

File: sql/.DS_Store
Comment (TomBillingtonUK) [inline sql/.DS_Store:1]: This file shouldn't be here

File: src/main/java/org/kainos/ea/api/temp.txt
Comment (TomBillingtonUK) [inline src/main/java/org/kainos/ea/api/temp.txt:1]: What's the purpose of this file?

File: src/main/java/org/kainos/ea/cli/temp.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+temp
```
Comment (TomBillingtonUK) [inline src/main/java/org/kainos/ea/cli/temp.txt]: Purpose of this file?

File: src/main/java/org/kainos/ea/client/temp.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+temp
```
Comment (TomBillingtonUK) [inline src/main/java/org/kainos/ea/client/temp.txt]: Purpose of this file?

File: src/main/java/org/kainos/ea/core/temp.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+temp
```
Comment (TomBillingtonUK) [inline src/main/java/org/kainos/ea/core/temp.txt]: Purpose of this file?

File: src/main/java/org/kainos/ea/db/temp.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+temp
```
Comment (TomBillingtonUK) [inline src/main/java/org/kainos/ea/db/temp.txt]: ?

File: src/main/java/org/kainos/ea/health/temp.txt [hunk 5]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (TomBillingtonUK) [inline src/main/java/org/kainos/ea/health/temp.txt]: ?

Comment (sammy-perry-kainos) [reply src/main/java/org/kainos/ea/health/temp.txt]: empty file here to ensure that folder exist

**Review by TomBillingtonUK — APPROVED** [review]

**Pull request merged**